src/styles/blocks/button.scss

```diff
@@ -0,0 +1,43 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  width: 100%;
+  height: 56px;
+  border-radius: 40px;
+  border: 3px solid $primary-font-color;
+
+  font-weight: $font-weight-bold;
+  font-size: 16px;
+  line-height: 27px;
+  text-decoration: none;
+  color: #fff;
+
+  background: $primary-font-color;
+
+  cursor: pointer;
+  transition: all 0.3s;
+
+  &:hover {
+    color: $primary-font-color;
+
+    background-color: transparent;
+  }
+
+  &:active {
+    border-color: #111;
+    color: #fff;
+
+    background-color: #111;
+  }
+
+  &--margin {
+    margin-bottom: 48px;
+
+    @include onTablet {
+      margin-bottom: 0;
+    }
+  }
```

Comment:
If you need a margin for block - set it to block where this button is located. Here using such kind of modifier looks strange.